multi-stage-onboarding-ask-and-show-to-pin.toml

```diff
@@ -0,0 +1,133 @@
+
+[experiment]
+
+segments = ["attributed", "non_attributed"]
+
+enrollment_period = 8
```

Comment:
Are you sure you want an 8-day enrollment period? Like the readme notes, this will cause Jetstream to sample nine "dates" and may throw off the rest of your analysis windows.

Reply:
I think so... i can manually adjust for this by +2'ing days to the end date, right?

Reply:
Yes, if that matches the deployment. I guess I think that you shouldn't override Experimenter here unless there was an error in Experimenter and that we should strive to make Experimenter correct -- this section is here so that you can break glass in an emergency, not for routine use.

What concerns do you have about leaving this set to 7? In general that's the value that should do what you expect for experiments with a one-week enrollment period -- i.e. "Tuesday to Tuesday"

Reply:
ah okay! I think the 7 addresses it, actually. 

I often see experiments where there is active enrollment happening on 8 total days, but it sounds like 7+1 should cover that.